src/app/window-handler.ts

```diff
@@ -4,6 +4,7 @@ import * as fs from 'fs';
 import * as path from 'path';
 import { format, parse } from 'url';
 
+import { ChildProcess, ExecException, execFile } from 'child_process';
```

Comment:
I think it is better to move this `screenShareIndicatorFrame` module to a separate file instead of handling everything in `window-handler.ts`

Reply:
No, it is not much code, and moving it would make it more complicated, as it ties in with the window handling. I think it is better to have it as part of window-handler

Reply:
@mattias-symphony Problem arrives when we want to write unit tests, that's the reason we try to componentize everything. My suggestion is same as @KiranNiranjan's as well.

But, let's also get @johankwarnmarksymphony's thoughts.

Reply:
Then I would suggest that at the point when we actually write the unit test, that would be a good time to refactor into a separate module.

Reply:
The screen share indicator frame is a separate windows program, so we already have it separated. I think we could wait with separating the java-script for now.

Reply:
@johankwarnmarksymphony Can you please explain why we'd want to postpone something we'd eventually do? We might as well do it now if we plan to do it in the future. Just my 2 cents.